x-pack/plugins/alerting/server/lib/create_abortable_es_client_factory.ts

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
This module looks so generic and useful, I wonder if there's a way we could get core to support it somehow - I imagine it could be useful in other situations outside of alerting.